core/ui/EditTemplate/body-toolbar-button.tid

```diff
@@ -35,22 +35,23 @@ title: $:/core/ui/EditTemplate/body/toolbar/button
   filter="[all[current]!has[dropdown]]"
   variable="no-dropdown"
 
-><$button
+><$set name=disabled filter={{!!condition-disabled}}><$button
 
   class="tc-btn-invisible $(buttonClasses)$"
   tooltip=<<tooltip-text>>
   actions={{!!actions}}
+  disabled=<<disabled>>
 
 ><span
 
-  data-tw-keyboard-shortcut={{!!shortcuts}}
+  data-tw-keyboard-shortcut={{{ [<disabled>compare:string:eq[yes]else{!!shortcuts}] +[trim[yes]] }}}
```

Comment:
This is tricky: ... 

 - `[<disabled>` .. gets the info from `$set` in line 38
 - `compare:string:eq[yes]` ... returns "yes" if the button should be disabled
   - if the tiddler or the field doesn't exist ... I till follow the "else" path
 - `else[!!shortcuts]` .. will set the data-tw... attribute, which is needed for the keyboard handler
 - `+[trim[yes]]` is needed to return an empty stirng, if the shortcut is disabled

Reply:
Just took a quick glance, but would this work, and if so be more legible?

`[<disabled>match[yes]then[]else{!!shortcuts}]`

Reply:
Seems to work. Thx for the input!